tests/test_models.py

```diff
@@ -106,6 +106,33 @@ async def test_model_get() -> None:
         await Movie.query().get()
 
 
+async def test_model_get_or_create() -> None:
```

Comment:
Is there any reason why this test is not added to the end of file?

I know right now it's not ideal and tests depend on each other, but we can fix that in a separate PR. Ideally each test should clean up database and start fresh.

That's why I don't think it's worth updating every test here. You can add yours to the end for now.

What do you think?

Reply:
now that you mention it, it makes more sense that this test is at the end of file.